pymodbus/events.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, **kwargs):
         self.listen = kwargs.get("listen", False)
         self.broadcast = kwargs.get("broadcast", False)
 
-    def encode(self):
+    def encode(self) -> str:
```

Comment:
I think a message is bytes ?

Reply:
Thanks, you are correct.  I'll annotate `pack_bitstring` and improve its docstring.

https://github.com/pymodbus-dev/pymodbus/blob/9c6acc385d2cb5362863cad740a90f693785f1e0/pymodbus/utilities.py#L109-L119

Reply:
Fixed in f5cd9fa817a5ec4fe19281c079d45e5307c3484e.

I think b212e79faf94379f59a0fb429230064d8c212ef6 is technically an API change (changing the name of an argument), although all current callers pass by value.

Reply:
you are right technically It is, but there are no changes in the app, so lets not mark it.

API_Changes.rst signals changes the user need to consider/modify (at least that is the intention).